.github/workflows/ci.yml

```diff
@@ -7,6 +7,9 @@ on:
     branches:
       - main
 
+# No permissions are required for this workflow
+permissions: {}
```

Comment:
is the usage of `{}` required? Is an empty value not sufficient, like usually used in `workflow_dispatch:`

Reply:
> You can use the following syntax to disable permissions for all of the available permissions:
> ```yaml
> permissions: {}
> ```

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/controlling-permissions-for-github_token

Reply:
thx for clarification